rules/TypeDeclaration/Rector/ClassMethod/AddReturnArrayDocblockBasedOnArrayMapRector.php

```diff
@@ -127,10 +127,10 @@ public function refactor(Node $node): null|Function_|ClassMethod
 
         $hasChanged = $this->phpDocTypeChanger->changeReturnType($node, $functionLikePhpDocInfo, $arrayType);
         if ($hasChanged) {
-            return null;
+            return $node;
```

Comment:
The logic here was reversed

Reply:
Thanks :+1:

Reply:
I wonder why the reversed logic did not make any unit test fail